src/languageServerInstaller.ts

```diff
@@ -153,11 +153,7 @@ function goArch(): string {
 	if (arch === 'x64') {
 		return 'amd64';
 	}
-	if (arch === 'arm64' && process.platform.toString() === 'darwin') {
-		// On Apple Silicon, install the amd64 version and rely on Rosetta2
-		// until a native build is available.
-		return 'amd64';
```

Comment:
Is this correct? Shouldn't it be `arm64` at least for Terraform 1.0.2?

Reply:
That is a good point! We could do version filtering here 🤔 

I will try patching that on Monday 👍🏻

Reply:
Actually these are LS versions, not Terraform versions and first LS for `darwin/arm64` was released [in February](https://github.com/hashicorp/terraform-ls/releases/tag/v0.14.0), and given that we don't really allow installing old versions & users are prompted to upgrade 24 hours after LS release I feel there's little benefit in adding that extra complexity.

In fact I would argue that we _want_ users to upgrade, so if returning an error for an _old_ version can help this, then I'd say it's even better?